test/ssl-tests/20-cert-select.conf.in

```diff
@@ -83,6 +85,20 @@ our @tests = (
         test   => {
             "ExpectedServerCertType" => "RSA",
             "ExpectedServerSignHash" => "SHA256",
+            "ExpectedServerSignType" => "RSA",
```

Comment:
Should the `ECDSA Signature Algorithm Selection` test above also have an `ExpectedServerSignType` line?